.kitchen.cloud.yml

```diff
@@ -36,7 +36,34 @@ platforms:
     driver:
       image_urn: MicrosoftWindowsServer:WindowsServer:Windows-Server-Technical-Preview:latest
     provisioner:
-      require_chef_omnibus: 11.18.6
+      require_chef_omnibus: 11.18.6    
+  - name: windows-2008r2-chef12.5.1
+    driver:
+      image_urn: MicrosoftWindowsServer:WindowsServer:2008-R2-SP1:latest
+      winrm_powershell_script: |-
+        winrm quickconfig -q
+        winrm set winrm/config/winrs '@{MaxMemoryPerShellMB="512"}'
+        winrm set winrm/config '@{MaxTimeoutms="1800000"}'
+        winrm set winrm/config/service '@{AllowUnencrypted="true"}'
+        winrm set winrm/config/service/auth '@{Basic="true"}'
+        netsh advfirewall firewall set rule name="Windows Remote Management (HTTP-In)" profile=public protocol=tcp localport=5985 remoteip=localsubnet new remoteip=any
+    provisioner:
+      require_chef_omnibus: 12.5.1
```

Comment:
Why did you change this to specifying 12.5.1 and not just letting it pick the latest 12 client?